infra/Pos/Slotting/MemState/Holder.hs

```diff
@@ -46,8 +45,18 @@ askSlottingTimestamp  = fst <$> askSlotting
 -- MonadSlotsData implementation
 ----------------------------------------------------------------------------
 
-instance MonadIO m =>
-         MonadSlotsData (SlottingHolder m) where
+data SlotsDataRedirectTag
+
+type SlotsDataRedirect =
+    Ether.TaggedTrans SlotsDataRedirectTag IdentityT
+
+runSlotsDataRedirect :: SlotsDataRedirect m a -> m a
+runSlotsDataRedirect = coerce
```

Comment:
Regarding the naming, the `Redirect` suffix is to remind that the transformer layer is nothing but an `IdentityT` to add an instance, it doesn't change the runtime representation or have a performance penalty.